pkg/minikube/cluster/cluster.go

```diff
@@ -192,6 +194,8 @@ type MachineConfig struct {
 func engineOptions(config MachineConfig) *engine.Options {
 	o := engine.Options{
 		Env:              config.DockerEnv,
+		AddRegistry:      config.AddRegistry,
+		BlockRegistry:    config.BlockRegistry,
```

Comment:
@sabre1041 you tested it manually right because current PR not even build minishift. It need some more time to thought about how we can pass those slice variable to provisioner.